src/OptionsResolver.php

```diff
@@ -113,7 +113,7 @@ public function getDefault($option)
      */
     public function getDefaults()
     {
-        return $this->defaults;
+        return $this->defaults; // maybe we get to SF to break open the parent?
     }
 
```

Comment:
I think they are required because the properties from the parent are private. I don't really like it but I didn't find a better solution for now.

> I wonder if maybe this can be changed in the SF class itself

I do plan to open feature/pull requests on symfony/options-resolver